index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+
+<html>
+	
+	<head>
+		<meta charset="utf-8">
+		<title>Relaxr Blog</title>
+		<link rel="stylesheet" type="text/css" href="css/normalize.css">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
```

Comment:
I'd recommend putting the the link to the Google fonts before your css/style.css file. Its not a hard requirement but because css/style.css depends upon the Google fonts, its generally a best practice to list things in a dependency order.